logstash/pipeline/logstash.conf

```diff
@@ -10,9 +11,19 @@ input {
 ## Add your filters / logstash plugins configuration here
 
 output {
-	elasticsearch {
-		hosts => "elasticsearch:9200"
-		user => elastic
-		password => "${ELASTIC_USER_PASSWORD:changeme}"
+	if [ENV] {
```

Comment:
if `ENV` != `test` maybe we should drop the `CI` key? Or just leave it with undefined values?